cpmonitor/static/js/main.js

```diff
@@ -0,0 +1,5 @@
+const progressbars = document.querySelectorAll("div.progress-bar");
+
+progressbars.forEach(pb => {
+  pb.style.width = pb.dataset.value + "%";
```

Comment:
Is this really better than setting the style in the HTML `style` directly?

Reply:
Generally, inline style is considered bad practice and a security issue. I guess that is the reason why someone else (wasn't me) added the no-inline-style task to #61 which in turn is the reason I implemented this.
If we want to use rule H021 (see below) and possibly a [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) it makes sense to be strict, as far as I am concerned. However, exceptions for specific cases would be possible, too, if you think that is better.

On the other hand, the question is what issue do you see with the js solution?

Reply:
Is more that it "feels" indirect and less elegant. But if there is a serious security issue, I'd go for less elegant... Thanx for the explanation.